### ISSUE TITLE
#.dtIewz------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#312]
Source: `<span class="CalciteA-sc-1pkt4xd-3 StyledTopNavLink-sc-10bmdec-5 czDMUZ dtIewz sc-kngDgl kWgnVs">Review</span>`
Element Location: .dtIewz
failureSummary: Fix any of the following:
  Element has insufficient color contrast of 3.71 (foreground color: #808080, background color: #f8f8f8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1Tags: [Ljava.lang.String;@45be7cd5
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
